docker-compose.yml

```diff
@@ -8,23 +8,18 @@ services:
       context: ./backend
     container_name: django_backend
     restart: unless-stopped
-    command: sh -c "
+    command: sh -c  "
       if [ \"$ENVIRONMENT\" = \"development\" ]; then
         python /app/backend/scripts/clear_images.py;
       fi;
+
       python manage.py wait_for_db &&
       python manage.py makemigrations &&
       python manage.py migrate &&
       python manage.py loaddata fixtures/superuser.json &&
       python manage.py loaddata fixtures/status_types.json &&
       python manage.py loaddata fixtures/topics.json &&
-      python manage.py populate_db \
-        --users 10 \
-        --orgs-per-user 1 \
-        --groups-per-org 1 \
-        --events-per-org 1 \
-        --resources-per-entity 1 \
-        --faq-entries-per-entity 1 &&
+      python manage.py populate_db --users 10 --orgs-per-user 1 --groups-per-org 1 --events-per-org 1 --resources-per-entity 1 --faq-entries-per-entity 1 &&
```

Comment:
Not sure why this was changed, to put all of the populate_db options on one line.

Reply:
Yes this would be nice to revert :)

Reply:
@Joshua-Tihabwangye , could you please revert these changes? Thanks!